src/main/java/seedu/saveit/model/issue/IssueFreqComparator.java

```diff
@@ -8,7 +8,26 @@
  * Comparator used to sort the Issues in search frequency order
  */
 public class IssueFreqComparator implements Comparator<Issue> {
+    /**
+     * Compare Issue a and b with their searchFrequency.
+     * If searchFrequency are the same, sort by created time.
+     */
     public int compare(Issue a, Issue b) {
-        return b.getFrequency().compare(a.getFrequency());
+        int freqDiff = b.getFrequency().compare(a.getFrequency());
```

Comment:
Same as here. I think UG and DG also need to be updated to state that the created time will be the back up order